### ISSUE TITLE
Windows 2019 evaluation key?

### ISSUE THREAD

**Issue by Larswa** [StefanScherer/packer-windows#146]
Hi @StefanScherer   Solid images. Love using them on Hyper-v

I wanted to testdrive the new ingress networking in Windows Server 2019, but i had some troubles with activating the evaluation version, to get the 180 days. Also tried giving it my 2019 standard MSDN license key, which didn't work either.

What iso are you using as a base for the two 2019 boxes?  I couldn't find the evaluation version of 2019 or Server 1809. Only a blogpost where it was stated that the eval version was not released yet. 

Did you manage to activate one of the new 2019 boxes? Eval or full license key?

best regards
Lars

**Comment by Larswa** [comment]
Wow .. weird that THAT is still available online. Seems like another bug it the mess that is the publishing of windows server 2019 / 1809.  :-)  

I's all over the evaluation site that the official 2019 eval will be published soon. So it would be awesome with an update of the 2019 boxes when that becomes available. Untill then, they will  only last 10 days as they are not the real eval versions.  

To make it even more of a tragedy, I decided to just hand install a couple 1809 boxes on my Hyper-v setup. But ... the 1809 on MSDN is a Volume Licensen and not a MAK (Multiple Activation Key) edition, and MSDN hands out MAK keys ... so that not even working. 

I went with an msdn 2019 server instead, and that was retail licensed. Which worked fine. So problem solved for now.

But heads up if anyone needs to run the 2019 boxes for more than 10 days. That won't fly.
